modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SkuResourceImpl.java

```diff
@@ -306,7 +307,7 @@ private Sku _addOrUpdateSKU(CPDefinition cpDefinition, Sku sku)
 			(BigDecimal)GetterUtil.get(sku.getPrice(), cpInstance.getPrice()),
 			(BigDecimal)GetterUtil.get(
 				sku.getPromoPrice(), cpInstance.getPromoPrice()),
-			serviceContext);
+			StringPool.BLANK, serviceContext);
```

Comment:
this looks wrong, this sku could hold data to create UOM and in this case is not manage like the productResourceImpl._updateNestedResources => the UOM and the prices will not be created.
Please adjust this as the ProductResourceImpl.
Thank you!

Reply:
also I believe this Util should do the same as here https://github.com/liferay/liferay-portal/blob/e9c93faeaa8c3ad54b80f37da4c5d816bb0c9d4f/modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SkuUnitOfMeasureResourceImpl.java#L196 and you could also use the util in place of this private method, so that we don't duplicate the code.